src/model/GroundHeatExchangerVertical.hpp

```diff
@@ -93,6 +95,8 @@ namespace model {
 
     bool isGFunctionReferenceRatioDefaulted() const;
 
+    ModelObject undisturbedGroundTemperatureModel() const;
```

Comment:
Similar question as above on the ctor, why a generic ModelObject? The name `undisturbedGroundTemperatureModel()` implies I'm going to get a thing of that type.

Reply:
See explanation above.

Reply:
got it.